src/test/store/receive-profile.test.js

```diff
@@ -1416,7 +1416,16 @@ describe('actions/receive-profile', function () {
       // Load a profile from the supplied mockFileOptions.
       const file = mockFile(mockFileOptions);
       const { dispatch, getState } = blankStore();
-      await dispatch(retrieveProfileFromFile(file, mockFileReader));
+      const browserConnectionStatus = await createBrowserConnection(
+        'Firefox/123.0'
+      );
+      const browserConnection =
+        browserConnectionStatus.status === 'ESTABLISHED'
+          ? browserConnectionStatus.browserConnection
+          : null;
```

Comment:
Ok, I converted this to an if + throw. An `expect(...).toBe(...)` is not enough to convince flow.

Reply:
Oh, nevermind, this test always ended up with `browserConnectionStatus.status === 'DENIED'`, because it mocks a WebChannel which always says "No such channel".
So I was able to simplify this whole setup: We no longer need to mock a WebChannel, we can just pass a null browserConnection. And we don't need to call createBrowserConnection.

Reply:
So this review comment was definitely warranted. I was confused about what the test was expecting!